INSTRUMENTS/KEITHLEY/KEITHLEY2400/KEITHLEY2400.py

```diff
@@ -1,9 +1,10 @@
-from flojoy import flojoy, OrderedPair, Vector
+from flojoy import flojoy, OrderedPair, Vector, node_initialization, NodeInitContainer
 import serial
 
 
 @flojoy(deps={"pyserial": "3.5"})
 def KEITHLEY2400(
+    init_container: NodeInitContainer,
     default: OrderedPair | Vector, comport: str = "/dev/ttyUSB0", baudrate: float = 9600
```

Comment:
Can remove the extra parameters here (comport, baudrate)

Reply:
Parameters removes